tide/wall/WallWindow.cpp

```diff
@@ -136,7 +137,16 @@ void WallWindow::exposeEvent( QExposeEvent* )
         _glContext->create();
 
     #if QT_VERSION >= 0x050500
+        // Call required to make QtGraphicalEffects work on this initial scene.
+        // Does the following internally:
+        //Q_D(QQuickRenderControl);
```

Comment:
Not really because I think there is no way to reproduce this behaviour, I can remove it